CampaignConfig/mdc2020_primary.cfg

```diff
@@ -25,7 +25,7 @@ nconcat = 1
 pdgcode = -11
 startmom = 0
 endmom=110
-bfield = "Offline/Mu2eG4/geom/bfgeom_DS70_no_tsu_ps_v01.txt"
+bfield = "Offline/Mu2eG4/geom/bfgeom_no_tsu_ps_v01.txt"
```

Comment:
I am not an expert on these scripts.   Was any production incorrectly run with the DS70 field map? 

Also, when you change the fieldmap, should you also change the particlename parameter?  In my mind those look coupled.

Reply:
no this was ran a while ago for the special camapign. Its not been used in any recent Production its been overridden in all cases in my local scripts